### PR TITLE
Add retry logic to inkuire fetch

### PR DESCRIPTION
**Pull request by pikinier20**
Even after increasing timeout limit, inkuire fetch is still flaky. This PR adds retry logic to stabilize the process

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -1331,16 +1331,32 @@ object Build {
...
-        val downloadProcess = (new java.net.URL(inkuireLink) #> inkuireDestinationFile).run()
-        val result: Future[Int] = Future(blocking(downloadProcess.exitValue()))
-        val res = try {
-          Await.result(result, duration.Duration(60, "sec"))
-        } catch {
-          case _: TimeoutException =>
-            downloadProcess.destroy()
-            throw new MessageOnlyException(s"Failed to fetch inkuire.js from $inkuireLink: Download timeout")
+
+        def tryFetch(retries: Int, timeout: Int): Unit = {
```
Comment (griggt) [inline project/Build.scala:1335]:
```suggestion
        def tryFetch(retries: Int, timeout: Duration): Unit = {
```

Consider using `Duration` here and updating the code accordingly.

File: project/Build.scala [hunk 2]
```diff
@@ -1331,16 +1331,32 @@ object Build {
...
+            Await.result(result, duration.Duration(timeout, "sec")) match {
+              case 0 =>
+              case res if retries > 0 =>
+                println(s"Failed to fetch inkuire.js from $inkuireLink: Error code $res. $retries retries left")
+                tryFetch(retries - 1, timeout)
+              case res => throw new MessageOnlyException(s"Failed to fetch inkuire.js from $inkuireLink: Error code $res")
+            }
+          } catch {
+            case e: TimeoutException =>
+              if(retries > 0) {
```
Comment (griggt) [inline project/Build.scala:1348]:
```suggestion
              if (retries > 0) {
```

File: project/Build.scala [hunk 3]
```diff
@@ -1331,16 +1331,32 @@ object Build {
...
+                println(s"Failed to fetch inkuire.js from $inkuireLink: Download timeout. $retries retries left")
+                tryFetch(retries - 1, timeout)
+              }
+              else {
+                downloadProcess.destroy()
```
Comment (griggt) [inline project/Build.scala:1353]: Do we want also want this for the `if` branch above when we catch TimeoutException?

**Review by griggt — APPROVED** [review]
A few minor comments and a question, otherwise LGTM.

File: project/Build.scala (hunk 1 above)
Comment (pikinier20) [reply project/Build.scala]: changed

File: project/Build.scala (hunk 3 above)
Comment (pikinier20) [reply project/Build.scala]: yes, we should destroy process in both branches

**Pull request merged**